cmd/trillian_log_server/main.go

```diff
@@ -43,7 +43,6 @@ import (
 	"github.com/google/trillian/storage"
 	"github.com/google/trillian/util/clock"
 	etcdutil "github.com/google/trillian/util/etcd"
-	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"google.golang.org/grpc"
 
 	// Register key ProtoHandlers
```

Comment:
The httpEndpoint flag says "Endpoint for HTTP metrics and **REST requests**" - should that second part be removed now?

Reply:
yep